src/display/editor/editor.js

```diff
@@ -124,7 +126,14 @@ class AnnotationEditor {
    * Initialize the l10n stuff for this type of editor.
    * @param {Object} _l10n
    */
-  static initialize(_l10n) {}
+  static initialize(_l10n) {
+    if (this._borderLineWidth !== -1) {
+      return;
+    }
+    const style = getComputedStyle(document.documentElement);
+    AnnotationEditor._borderLineWidth =
```

Comment:
Nit: Since you're in a static method, can't you access a static property as follows?
```suggestion
    this._borderLineWidth =
```

Reply:
Unfortunately, it doesn't work as I expected.
When a subclass is calling the function, the property is modified in the subclass but not in the parent class.
So we've to use `AnnotationEditor.`.